src/javascripts/helpers/data/globals.js

```diff
@@ -0,0 +1,36 @@
+const currentDashboard = [];
+const currentUser = [];
+
+const getUser = () => currentUser;
+
+const setUser = (user) => {
+  if (getUser().length === 0) {
+    currentUser.push(user);
+  } else {
+    currentUser.pop();
+    currentUser.push(user);
+  }
+};
+
+const clearUser = () => {
+  currentUser.splice(0, currentUser.length);
```

Comment:
You could probably just reassign currentUser to an empty array here. Did you try that?

Reply:
I think that weird syntax was trying to solve an eslint issue, and the fact that currentUser is a constant, it can't be reassigned I think, but constant arrays are mutable though.

Reply:
Ah. Yes reassignment is weird so, you can change `const` to `let` and that should do the trick. I am going to go ahead and approve this, but please make that change. Create a ticket for it to keep track of all the work you are doing. Excellent job!